x-pack/plugins/security_solution/public/common/components/navigation/helpers.ts

```diff
@@ -57,6 +59,13 @@ export const getSearch = (tab: SearchNavTab, urlState: UrlState): string => {
           } else {
             urlStateToReplace = timeline;
           }
+        } else if (urlKey === CONSTANTS.detailPanel && urlState[CONSTANTS.detailPanel] != null) {
```

Comment:
The same comment about unit testing here: please, open the follow up issue or do some changes if you think is reasonable in the current PR.